Samsung/a71n/res/values/bools.xml

```diff
@@ -0,0 +1,24 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+    <bool name="config_showNavigationBar">true</bool>
+    <bool name="config_automatic_brightness_available">true</bool>
+    <bool name="config_useDevInputEventForAudioJack">false</bool>
+    <bool name="config_bluetooth_le_peripheral_mode_supported">false</bool>
+    <bool name="config_wifi_dual_band_support">false</bool>
+    <bool name="config_wifi_background_scan_support">true</bool>
+    <bool name="skip_restoring_network_selection">true</bool>
+    <bool name="config_hotswapCapable">true</bool>
+    <bool name="config_wifiDisplaySupportsProtectedBuffers">false</bool>
+    <bool name="config_wifi_batched_scan_supported">false</bool>
+    <bool name="config_speed_up_audio_on_mt_calls">false</bool>
+    <bool name="config_device_volte_available">true</bool>
+    <bool name="config_device_vt_available">true</bool>
+    <bool name="config_device_wfc_ims_available">true</bool>
+    <bool name="config_carrier_volte_available">true</bool>
+    <bool name="config_switch_phone_on_voice_reg_state_change">true</bool>
+    <bool name="config_supportAudioSourceUnprocessed">false</bool>
```

Comment:
sounds weird, I'm pretty sure this is a requirement for all devices since Nougat, so this device should support it

Reply:
I've checked the /vendor/overlay. the values are false :(